src/dotenv/cli.py

```diff
@@ -102,11 +113,17 @@ def get(ctx: click.Context, key: Any) -> None:
 @click.argument('key', required=True)
 def unset(ctx: click.Context, key: Any) -> None:
     '''Removes the given key.'''
-    file = ctx.obj['FILE']
     quote = ctx.obj['QUOTE']
-    success, key = unset_key(file, key, quote)
-    if success:
-        click.echo("Successfully removed %s" % key)
+
+    global_success = False
+    success_files = []
+    for file in ctx.obj['FILES']:
+        success, key = unset_key(file, key, quote)
+        if success:
+            global_success = True
+            success_files.append(file)
+    if global_success:
+        click.echo("Successfully removed %s from %s" % (key, success_files[0] if len(success_files) == 1 else success_files))
```

Comment:
this line has a formatting issue but I don't know how to fix it 🤔

Reply:
Fixed by [2b3a28b](https://github.com/theskumar/python-dotenv/pull/424/commits/2b3a28be6ee47c6e4780c5df6196642bf2e5e894)